.github/workflows/dependency-review.yml

```diff
@@ -35,3 +35,6 @@ jobs:
 
       - name: Review Dependencies
         uses: actions/dependency-review-action@ce3cf9537a52e8119d91fd484ab5b8a807627bf8 # v4.6.0
+        with:
+          allow-ghsas:
+            - GHSA-67mh-4wv8-2f99 # esbuild@0.19.2 - transitive dependency of @opennextjs/cloudflare
```

Comment:
I think if you don't include this it'll error on this PR, but not future ones, right? The action only checks _changed_ dependencies, and it's not a required check.

Reply:
yes, sorry I think you're totally right!

what are you suggesting then? to remove this change and just ignore the failure in this PR?

(PS: I don't think there a specific reason for the open-next adapter to use esbuild 0.19.2, so I need to check if we can just bump it there 👍)

Reply:
> what are you suggesting then? to remove this change and just ignore the failure in this PR?

Yep!

Reply:
ok thanks 🙂 , I've reverted the change 🙂